.github/workflows/test.yml

```diff
@@ -0,0 +1,64 @@
+name: test
+
+on:
+  push:
+    branches:
+      # - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.6"
+          - "2.7"
+          - "3.0"
+          - "3.1"
+          - "3.2"
+          - "3.3"
+        rails:
+          - "5.2"
+          - "6.0"
+          - "6.1"
+          - "7.0"
+        cassandra:
+          - "2.2"
+          - "3.0"
+          - "3.11"
+        exclude:
+          - ruby: "2.6"
+            rails: "7.0"
+          - ruby: "3.0"
+            rails: "5.2"
+          - ruby: "3.1"
+            rails: "5.2"
+          - ruby: "3.2"
+            rails: "5.2"
+          - ruby: "3.3"
+            rails: "5.2"
```

Comment:
```suggestion
        ruby:
          - 2.7
          - 3.0
          - 3.1
          - 3.2
          - 3.3
        rails:
          - 6.0
          - 6.1
          - 7.0
        cassandra:
          - 2.2
          - 3.0
          - 3.11
```
We can drop 2.6 and 5.2 since we no longer use either and they're well passed EOL

Reply:
Added in latest push